### ISSUE TITLE
Fix padding of workspace thumbnails in overview

### ISSUE THREAD

**Issue by mzur** [mzur/gnome-shell-wsmatrix#181]
The workspace thumbnails in the overview should have vertical padding (same as horizontal). Maybe related to #177

**Comment by ebeem** [comment]
You're right, I think it's related to #177 
The reason behind that is because we actually have lists in the `WorkspaceSwitcherPopupList` equal to the amount of rows we have in `wsmatrix`. Long story with the limitation faced while implementing a grid in the `WorkspaceSwitcherPopupList`.

I think to fix this, we need to build a `St.ScrollView` to contain a single list which has all the workspaces.
In other words, `_lists` should be one list instead of a array of lists
https://github.com/mzur/gnome-shell-wsmatrix/blob/64c3895c0507718bf5a77ac2a04cabb3e793fbba/wsmatrix%40martin.zurowietz.de/workspacePopup/workspaceSwitcherPopupList.js#L29-L47
